superset-frontend/src/components/Form/LabeledErrorBoundInput.tsx

```diff
@@ -49,18 +49,17 @@ const alertIconStyles = (theme: SupersetTheme, hasError: boolean) => css`
   ${hasError &&
   `.ant-form-item-control-input-content {
       position: relative;
-
       &:after {
         content: ' ';
         display: inline-block;
         background: ${theme.colors.error.base};
-        mask: url('/images/icons/error.svg');
+        mask: url(${errorIcon});
         mask-size: cover;
         width: ${theme.gridUnit * 4}px;
         height: ${theme.gridUnit * 4}px;
         position: absolute;
-        right: 7px;
-        top: 15px;
+        right: 2%;
```

Comment:
Could you change `right` and `top` to have static values with `px` instead of `%`? The field size shouldn't be changing so a static value should be fine here. Could you also change the value to use `theme.gridUnit`? Each gridUnit is 4, so if you wanted a value of 16px you'd do `${theme.gridUnit * 4}px`